Packages/com.unity.inputsystem/Documentation~/ActionBindings.md

```diff
@@ -927,6 +940,29 @@ Press(Keyboard.current.leftShiftKey);
 Press(keyboard.bKey);
 ```
 
+If an application needs even more control over when input events are consumed, it is possible to completely take over the handling of events by using the [`CallbackContext.HandleEvent`](api/UnityEngine.InputSystem.InputAction.CallbackContext.html#UnityEngine_InputSystem_InputAction_CallbackContext_HandleEvent) method. In the following example, event consumption is turned off for the `Shift+B` binding, and the performed handler only handles the input if the 'inventoryOpen' flag is true. The `B` binding then decides whether to perform some actions based on whether the input has been handled or not.
+
+```CSharp
+bAction.AddBinding("<Keyboard>/b");
+shiftbAction.AddCompositeBinding("OneModifier(handleInputEvents=false)")
```

Comment:
In this example "consume input events" is off, right?

Reply:
It's actually on. I modified the example to include the same configuration code as the example above it.